CRM/Core/Form/EntityFormTrait.php

```diff
@@ -30,6 +30,19 @@ trait CRM_Core_Form_EntityFormTrait {
    */
   protected $_entitySubTypeId = NULL;
 
+  /**
+   * Message to show when asking if the user wants to delete the object.
+   *
+   * @var string
+   */
+  protected $deleteMessage;
+
+  /**
+   * Various form meta data
+   * @var array
```

Comment:
Not a biggie but this commend block is a bit inconsistent - it the one above has a full stop & line break after (which I think is inline with the drupal standard bits we don't enforce)

Reply:
Note this is covered in https://github.com/civicrm/civicrm-core/pull/25802